src/Starlight/Draw/Body.hs

```diff
@@ -57,17 +58,20 @@ drawBody
      , Has (Lift IO) sig m
      , Has Profile sig m
      , Has (Reader Drawable) sig m
+     , Has (Reader (System Body.StateVectors)) sig m
      , Has (Reader View) sig m
      )
   => Body.StateVectors
   -> m ()
-drawBody Body.StateVectors{ body = Body.Body{ radius = Kilo (Metres r), colour }, transform, actor = Actor{ rotation } } = measure "bodies" . UI.using getDrawable $ do
+drawBody Body.StateVectors{ body = Body.Body{ radius, colour }, transform, actor = Actor{ rotation } } = measure "body" . UI.using getDrawable $ do
   vs@View{ focus } <- ask
+  systemTrans <- asks (systemTrans @Body.StateVectors)
   matrix_
     ?=  scaleToViewZoomed vs
-    !*! translated3 (ext (negated (unP focus)) 0) -- transform to the origin
+    !*! systemTrans
```

Comment:
Since the system scale is no longer inherent in body positions, we apply it explicitly when drawing scene objects instead.